frontend/src/libs/constants/chat-mock-data.constant.ts

```diff
@@ -0,0 +1,14 @@
+import { UserRole } from '#libs/enums/enums.js';
```

Comment:
let's remove this from global constants and move this in component constatns, where it's used